ktor-generator-core/src/main/kotlin/io/ktor/generator/core/features/EngineFeatures.kt

```diff
@@ -0,0 +1,33 @@
+package io.ktor.generator.core.features
+
+/**
+ * Fake features for templating (see [NettyEngine], [JettyEngine], [TomcatEngine], [CioEngine]).
+ *
+ * Adds corresponding engine dependency to the list of build system dependencies and allows generating main class name
+ * based on selected engine.
+ * */
+open class EngineFeature(val engineName: String) : KtorFeature {
+    override val id = engineName
+    override val name = engineName
+    override val description = null
```

Comment:
Do you mean abstract classes with null value overrides, that we can inherit?
It can be the case but in fact these are only 2 properties and only 2 classes that inherit `KtorFeature` so not sure if we will gain any readability from this.

Reply:
I meant to have separate objects with description and without it.